packages/tradershub/src/components/CurrencySwitcher/CurrencySwitcher.tsx

```diff
@@ -11,6 +12,8 @@ type AccountActionButtonProps = {
 };
 
 const AccountActionButton = ({ balance, isDemo }: AccountActionButtonProps) => {
+    const history = useHistory();
+    const { mutate: resetVirtualBalance } = useResetVirtualBalance();
```

Comment:
```suggestion
    const { mutate: resetVirtualBalance } = useResetVirtualBalance();
    const canReset = isDemo && balance !== 10000;
```
use this `canReset` to replace similar checks in line 18 and line 29. Also i think u need to add a disable style based on this boolean as well

Reply:
We wont be seeing the reset balance button if the balance is 10000

Reply:
can can

Reply:
wokehz onz